addons/project-folders/userscript.js

```diff
@@ -0,0 +1,190 @@
+export default async function ({ addon, global, console, msg }) {
+  let projectColumns;
+  projectColumns = await addon.tab.waitForElement("ul.media-list", {
+    markAsSeen: true,
+  });
+
+  let foldersArray;
+  const projectTab = document.querySelectorAll("li.first")[0];
+  projectTab.childNodes[0].onclick = async () => {
+    projectColumns = await addon.tab.waitForElement("ul.media-list", {
+      markAsSeen: true,
+    });
+    createFolderAreaAndButton();
+    foldersArray = await load();
+  };
+
+  if (document.querySelectorAll("div.folders-container")[0] === undefined) {
+    createFolderAreaAndButton();
+    foldersArray = await load();
+  }
+
+  async function getProjectDetails(projectID) {
+    const token = await addon.auth.fetchXToken();
+    const projectDetails = await (
+      await fetch(`https://api.scratch.mit.edu/projects/${projectID}/`, {
+        headers: {
+          "content-type": "application/json",
+          "x-csrftoken": addon.auth.crsfToken,
+          "x-token": token,
+        },
+      })
+    ).json();
+
+    return projectDetails;
+  }
+
+  async function load() {
+    const folders = [];
+
+    const folderDiv = document.querySelectorAll(".folders")[0];
+
+    const noFolderSpan = document.createElement("span");
+    noFolderSpan.textContent = msg("loading");
+    folderDiv.appendChild(noFolderSpan);
+
+    const tempBR = document.createElement("br");
+    folderDiv.appendChild(tempBR);
+
+    const loader = document.createElement("progress");
+    loader.setAttribute("value", "0");
+    loader.setAttribute("max", "100");
+    folderDiv.appendChild(loader);
+
+    for (let i = 0; i < projectColumns.childNodes.length - 2; i++) {
+      loader.setAttribute("value", (i / (projectColumns.childNodes.length - 2)) * 100);
+
+      const link = projectColumns.childNodes[i + 2].childNodes[1].childNodes[3].childNodes[1].childNodes[0].href;
+      const projectID = link.replace("https://scratch.mit.edu/projects/", "").replace("/", "");
+      const projectDetails = await getProjectDetails(projectID);
+      let instructions = projectDetails.instructions;
+
+      if (instructions.includes("#_")) {
```

Comment:
```suggestion
      if (!instructions.includes("#_")) continue;
```
Just remember to remove the closing brace.